tests/cases/stream_test.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+###############################################################################
+#  Copyright 2014 Kitware Inc.
+#
+#  Licensed under the Apache License, Version 2.0 ( the "License" );
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+###############################################################################
+
+import os
+import requests
+import time
+
+from .. import base
+from girder import config
+from girder.api import access
+from girder.api.rest import Resource, iterBody
+
+_chunks = []
+os.environ['GIRDER_PORT'] = os.environ.get('GIRDER_TEST_PORT', '20200')
+config.loadConfig()  # Must reload config to pickup correct port
+
+
+def setUpModule():
+    server = base.startServer(mock=False)
+    server.root.api.v1.stream_test = StreamTestResource()
+
+
+def tearDownModule():
+    base.stopServer()
+
+
+class StreamTestResource(Resource):
+    def __init__(self):
+        super(StreamTestResource, self).__init__()
+        self.resourceName = 'stream_test'
+        self.route('POST', ('input_stream',), self.inputStream)
+
+    @access.public
+    def inputStream(self, params):
+        for chunk in iterBody(8):  # Read body 8 bytes at a time
```

Comment:
Easy number to demonstrate that we are indeed receiving the correct chunk lengths. This number is essentially test fixture data, it correlates to our other arbitrary fixture data which is the dummy chunks we are sending in each case.